### PR TITLE
Feature/cassandra provider

### PR DESCRIPTION
**Pull request by erenming**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
In order to interact with Cassandra

#### Which issue(s) this PR fixes:

Fixes #

#### Reivewers List:
@recallsong @liuhaoyang 

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation usage docs, etc.:

```docs

```

### CODE REVIEW

File: providers/cassandra/cassandra.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+	Security bool          `file:"security" env:"CASSANDRA_SECURITY_ENABLE" default:"false" desc:"security"`
+	Username string        `file:"username" env:"CASSANDRA_SECURITY_USERNAME" default:"" desc:"username"`
+	Password string        `file:"password" env:"CASSANDRA_SECURITY_PASSWORD" default:"" desc:"password"`
+	Timeout  time.Duration `file:"timeout" env:"CASSANDRA_TIMEOUT" default:"3s" desc:"session timeout"`
+}
+
+type define struct{}
+
+func (d *define) Service() []string   { return []string{"cassandra"} }
+func (d *define) Summary() string     { return "cassandra" }
```
Comment (liuhaoyang) [inline providers/cassandra/cassandra.go:75]: Use `Services` to describe dependencies, not `Service`

**Review by liuhaoyang — CHANGES_REQUESTED** [review]
Provide a example program, please !

File: providers/cassandra/cassandra.go (hunk 1 above)
Comment (erenming) [reply providers/cassandra/cassandra.go:75]: We may need a contributing guideline

**Comment by erenming** [comment]
> Provide a example program, please !

done

**Review by liuhaoyang — APPROVED** [review]

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**